client/blocks/term-form-dialog/README.md

```diff
@@ -0,0 +1,63 @@
+TermFormDialog
+==============
+
+The `TermFormDialog` component renders a dialog that allows to create Terms.
+
+## Usage
+
+```jsx
+import TermFormDialog from 'blocks/term-form-dialog';
+
+<TermFormDialog showDialog={ true } taxonomy="category" onClose={callback} postType="post" />
+```
+
+## Props
+
+
+### `postType`
+
+<table>
+	<tr><th>Type</th><td>String</td></tr>
+	<tr><th>Required</th><td>Yes</td></tr>
+</table>
+
+The type of post that uses this taxonomy.
+
+### `taxonomy`
+
+<table>
+	<tr><th>Type</th><td>String</td></tr>
+	<tr><th>Required</th><td>Yes</td></tr>
+</table>
+
+The type of taxonomy to add.
+
+### `showDialog`
+
+<table>
+	<tr><th>Type</th><td>Boolean</td></tr>
+	<tr><th>Required</th><td>No</td></tr>
+	<tr><th>Default</th><td><code>false</code></td></tr>
+</table>
+
+Whether the dialog is shown/hidden
+
+### `onClose`
+
+<table>
+	<tr><th>Type</th><td>Function</td></tr>
+	<tr><th>Required</th><td>No</td></tr>
+	<tr><th>Default</th><td><code>() => {}</code></td></tr>
+</table>
+
+A function invoked when the dialog is closed.
```

Comment:
Is there an argument passed to this function - or what happens if the add operation fails.